app/web/features/auth/locales/en.json

```diff
@@ -330,8 +330,20 @@
       "enabled_message": "You <1>are currently</1> verified with Strong Verification.",
       "disabled_message": "You <1>are currently not</1> verified with Strong Verification."
     },
-    "complete_in_console": "You can complete Strong Verification <2>on this page</2>. We are working on improving the flow and implementing it fully in the platform.",
-    "go_to_button": "Go to Strong Verification settings"
+    "start_button": "Start Strong Verification",
+    "begin_button": "Complete Strong Verification with Iris ID",
+    "information_text1": "You will need a biometric passport and an NFC-capable Apple iPhone or Android phone.",
+    "information_text2": "By clicking the button below, you will be taken to the Iris ID website to complete the passport verification process.",
+    "information_text3": "You can read more about how we and Iris ID process and store your data in our <2>Terms of Service</2>. As per the Iris ID <5>Privacy policy</5>, they will delete your personal information within 24 hours. You can request for this Strong Verification data (except issuing country, expiry date, and the last three digits of the document number) to be deleted from our servers at any time with the click of a button.",
+    "complete_title": "We are processing your Strong Verification attempt",
+    "complete_message1": "Your passport is being validated. If it matches your gender and date of birth, you will soon receive an email within approximately 15 minutes.",
```

Comment:
The 15 minute wait time seems really long. Why does it take so long to process?

Reply:
I actually realized I fixed that, changed to "a few minutes"